admin/class-yoast-notification-center.php

```diff
@@ -146,6 +151,42 @@ public static function maybe_dismiss_notification( Yoast_Notification $notificat
 		return self::dismiss_notification( $notification, $meta_value );
 	}
 
+	/**
+	 * Dismisses a notification.
+	 *
+	 * @param Yoast_Notification $notification Notification to dismiss.
+	 * @param string             $meta_value   Value to save in the dismissal.
+	 *
+	 * @return bool True if dismissed, false otherwise.
+	 */
+	public static function dismiss_notification( Yoast_Notification $notification, $meta_value = 'seen' ) {
+		// Dismiss notification.
+		return update_user_option( get_current_user_id(), $notification->get_dismissal_key(), $meta_value ) !== false;
+	}
+
+	/**
+	 * Restores a notification.
+	 *
+	 * @param Yoast_Notification $notification Notification to restore.
+	 *
+	 * @return bool True if restored, false otherwise.
+	 */
+	public static function restore_notification( Yoast_Notification $notification ) {
+
+		$user_id       = get_current_user_id();
+		$dismissal_key = $notification->get_dismissal_key();
+
+		// Restore notification.
+		$restored = delete_user_option( $user_id, $dismissal_key );
+
+		// Delete unprefixed user meta too for backward-compatibility.
+		if ( metadata_exists( 'user', $user_id, $dismissal_key ) ) {
+			$restored = delete_user_meta( $user_id, $dismissal_key ) && $restored;
```

Comment:
This line seems to be missing test coverage.

Reply:
I added an extra test for this.